spec/decorators/author_decorator_spec.rb

```diff
@@ -9,8 +9,8 @@
     its(:reverse_name) { should eq("oob") }
 
     its(:form) {
-      should match %r|^<form accept-charset="UTF-8" action="/authors/#{author.id}" class="edit_author" id="edit_author_#{author.id}" method="post">|
-      should match %r|<input id="author_name" name="author\[name\]" type="text" value="boo" />|
+      should match %r|^<form class=\"edit_author\" id=\"edit_author_4\" action=\"/authors/4\" accept-charset=\"UTF-8\" method=\"post\">|
```

Comment:
Thank you for this PR, but test is broken.  I think it will fix the following changes:

```diff
-      should match %r|^<form class=\"edit_author\" id=\"edit_author_4\" action=\"/authors/4\" accept-charset=\"UTF-8\" method=\"post\">|
+      should match %r|^<form class=\"edit_author\" id=\"edit_author_#{author.id}\" action=\"/authors/#{author.id}\" accept-charset=\"UTF-8\" method=\"post\">|
```